### DISCUSSION TITLE
i3 is not opening

### DISCUSSION THREAD

**Discussion by banoni-dev** [i3/i3#5812] [Q&A]
## i3 is not opening with the superuser

<pre>
[x] Bug
[ ] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

https://github.com/i3/i3/assets/77343627/d9879628-9fe7-47df-898e-db8f4f0a4655

## Current Behavior

i am trying to login using the i3WM but nothing happens and i am obliged to login with the ubuntu option.
i run i3 in my terminal and i got this

![image](https://github.com/i3/i3/assets/77343627/53602dfc-b244-4480-8714-02f1de119a19)

when i created new user and select i3  it automaticly works without installing or configure anything

i tried to remove i3 but i couldn't he is still there 

## Environment

Output of `i3 --moreversion 2>&-`:

i3 version: 
![image](https://github.com/i3/i3/assets/77343627/ed3583be-9cb9-4a49-9efd-d6847456d42c)

<pre>
- Linux Distribution & Version: Ubuntu 22.04
</pre>

**Comment by banoni-dev** [comment]
> I don’t see a version number. Could you please copy & paste the output of `i3 --version` into this issue?

![image](https://github.com/i3/i3/assets/77343627/23b7cb7d-8938-4a15-b63f-9fe5b8c14de7)

in the link provided https://i3wm.org/docs/debugging.html? it requires  version >= 4.23 and there is an example  of version 4.7 and it is not existing yet. i am really confused. how to upgrade my i3 version if this is required and what are the other steps. the problem is i3 is working when i choose another user but for tthe superuser is not working

**Reply by orestisfl** [reply]
4.23 is the newest version, you are okay on that front

**Comment by banoni-dev** [comment]
> I don’t see a version number. Could you please copy & paste the output of `i3 --version` into this issue?

i am getting these errors now when i run i3 

![image](https://github.com/i3/i3/assets/77343627/83611fd6-8f45-4b10-86da-7aaf8314d60f)

**Reply by orestisfl** [reply]
I suggest starting with the default config and modifying from there

Otherwise, put this line at the top of your config file
```
# i3 config file (v4)
```

**Comment by orestisfl** [comment]
You can't just run i3 from the terminal in your existing session, you need to start a new session with i3 as the window manager.

How to do that depends a bit on your distro but the Arch wiki will be a good place to start understanding your options: https://wiki.archlinux.org/title/i3

**Reply by banoni-dev** [reply]
i am using ubuntu 22.04
i tried to uninstall it and restart but i can't it still even even i remove every file or folder with i3 in my system. i really i don't know what to do. he is working on the other user but want it working in the superuser so can i use all my files .... have you any solution to remove it completely from  my system and i'll try to reinstall or there are any other good alternatives for i3

**Reply by orestisfl** [reply]
You can `rm -rf ~/.config/i3` and try again

**Comment by christian-grothe** [comment]
Hey! Is this solved? Currently running into the same issue. I copied the default config into .config/i3 but still am not able to login

**Reply by banoni-dev** [reply]
no i switched to hyprland
